include/seccomp-syscalls.h

```diff
@@ -1286,6 +1287,12 @@
 
 #define __SNR_openat			__NR_openat
 
+#ifdef __NR_openat2
+#define __SNR_openat2			__NR_openat2
+#else
+#define __SNR_openat2			__PNR_openat2
+#endif
```

Comment:
See my comment above, this can be simplified since we don't need `__PNR_openat2`.